### DISCUSSION TITLE
expense-shell backend.sh

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1628] [Q&A] [answered]
![Screenshot 2023-11-22 170048](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/ccdf087e-c74e-4848-83c7-7db9d095bb09)
Load schema Failure, 
 Ok i got it

**Comment by sreedharm07** [comment, marked as answer]
@palsachennaiah 
password is ExpenseApp@1
but you given 
**ExpeseApp@1**
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/d1f1f4ac-c539-4ab2-8172-a9e504e34246)

**Reply by palsachennaiah** [reply]
Ok i got it,
